docs/query-api.md

```diff
@@ -0,0 +1,77 @@
+# Query API with Server-sent Events (SSE)
+
+All Timeplus functions is available through RESTful API. To support real-time query result pushed from server to client side, there are two popular solutions, [websocket](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) and [server-sent events](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events). Timeplus is now leveraging sever-sent events to push real-time query result to the client.
```

Comment:
- is to are
- RESTful API to REST API (to keep it consistent with rest of the docs)